src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/Snapshot/TargetedDependenciesSnapshotTests.cs

```diff
@@ -573,6 +573,47 @@ public void TFromChanges_RemovedAndAddedChanges()
             Assert.True(snapshot.DependenciesWorld.ContainsKey(@"tfm1\xxx\addeddependency3"));
         }
 
+        /// <summary>
+        /// Added because circular dependencies can cause stack overflows
+        /// https://github.com/dotnet/project-system/pull/3474
```

Comment:
Link to [https://github.com/dotnet/project-system/issues/3374](https://github.com/dotnet/project-system/issues/3374) instead please

Reply:
Done